libs/journeys/ui/src/components/TextResponse/utils/getTextResponseValues/getTextResponseValues.ts

```diff
@@ -11,9 +11,10 @@ export interface TextResponseStrings {
 export function getTextResponseValues(
   textResponseStrings: TextResponseStrings,
   journeyCustomizationFields: JourneyCustomizationField[],
-  variant: JourneyProviderContext['variant']
+  variant: JourneyProviderContext['variant'],
+  journeyIsTemplate: boolean = false
```

Comment:
```suggestion
  journeyIsTemplate?: boolean = false 
```

Make field nullable | undefined as well, that way, when u consume function you can just pass in journey?.template without having to null check it.